app/controllers/events/view/tickets/orders.js

```diff
@@ -0,0 +1,46 @@
+import Controller from '@ember/controller';
+import { run } from '@ember/runloop';
+
+export default Controller.extend({
+  isLoadingcsv: false,
+
+  isLoadingpdf: false,
+
+  actions: {
+    export(mode) {
+      this.set(`isLoading${  mode}`, true);
+      this.get('loader')
+        .load(`/events/${this.get('model.id')}/export/orders/${  mode}`)
+        .then(exportJobInfo => {
+          this.requestLoop(exportJobInfo);
+        })
+        .catch(() => {
+          this.set(`isLoading${  mode}`, false);
+          this.get('notify').error(this.get('l10n').t('Unexpected error occurred.'));
+        });
+    }
+  },
+  requestLoop(exportJobInfo) {
+    run.later(() => {
+      this.get('loader')
+        .load(exportJobInfo.task_url, { withoutPrefix: true })
+        .then(exportJobStatus => {
+          if (exportJobStatus.state === 'SUCCESS') {
+            window.open(exportJobStatus.result.download_url);
```

Comment:
@mayank8318 The CSV export is working fine, but using `window.open` results in a pop-up which chrome and maybe other browsers block by default. This might confuse the users, make it so that once it's ready the download starts.